<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_LVS</class>
 <widget class="QWidget" name="Widget_LVS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="lvsTab">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AllEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>40</number>
   </attribute>
   <column>
    <property name="text">
     <string>电流下限A</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>电流上限A</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>功率下限W</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>功率上限W</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>时间S</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>621</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lable_volt">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>91</horstretch>
        <verstretch>41</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>外部电压：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="LVS_Volt">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>61</horstretch>
        <verstretch>41</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>41</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>255.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_fre">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>61</horstretch>
        <verstretch>41</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>频率：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="LVS_Fre">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>41</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>61</width>
        <height>41</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>99.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_channel">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>通道：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="LVS_channel">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>41</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>3.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lable_Turn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>89</horstretch>
        <verstretch>41</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>转向：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="combConf_Turn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	color: rgb(255, 255, 255);
    border: 1px solid #32435E;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    background:qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #080B10,stop:1 #212C3F);
}
QComboBox::down-arrow {
    image: url(:image/downarrow.png);
}
QComboBox::drop-down {

    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 25px;
    border-left-width: 1px;
    border-left-color: 32435E;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}

QComboBox QAbstractItemView {
	color: rgb(255, 255, 255);
    border: 2px solid #32435E;
    selection-background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #506B79,stop: 1.0 #0D95A6);
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #1B2534, stop: 0.4 #010101,stop: 0.5 #000101, stop: 1.0 #1F2B3C);
}</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>反转</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>正转</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="toolButton_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>98</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{border:0px;}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    border: 1px solid white;
    border-color: #447684;
    color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="radio_Pwr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton
{
color: rgb(255, 255, 255);
}
 QRadioButton::indicator {
	color: rgb(255, 255, 255);
image: url(:/image/053.png);
      width: 45px;
      height: 35px;
  }
QRadioButton::indicator:checked {
image: url(:/image/051.png)
;}&quot;);
</string>
    </property>
    <property name="text">
     <string>功率判定</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_Turn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton
{
color: rgb(255, 255, 255);
}
 QRadioButton::indicator {
	color: rgb(255, 255, 255);
image: url(:/image/053.png);
      width: 45px;
      height: 35px;
  }
QRadioButton::indicator:checked {
image: url(:/image/051.png)
;}&quot;);


</string>
    </property>
    <property name="text">
     <string>转向判定</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>toolButton_4</tabstop>
  <tabstop>LVS_Fre</tabstop>
  <tabstop>LVS_Volt</tabstop>
  <tabstop>LVS_channel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
